test/app-media-audio.html

```diff
@@ -15,12 +15,9 @@
 
     <title>app-media-audio test</title>
 
-    <script src="../../../@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
+    <script src="../node_modules/@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
```

Comment:
Should we be changing these to the loader? (Here and also in other test files.) This is something I was wondering about too.

Reply:
(from offline) No, we're going to leave these pointing to the bundle.